File: sqlInsert.php
<?php
$servername = "localhost";
$username = "root";
$password = "password";
$dbname = "school";


//create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// check connection
if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

//create variable and load/save SQL staement
$sql = "INSERT INTO students (name, age, gradeLevel)VALUES ('John', 14, 8)";
//taking our statement (sql) passing the query into the object because the object-
//handle all the back end
if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?> 

File: sqlInsert.py
import pymysql
 
db = pymysql.connect(host="localhost", user="root", passwd="password", db="school")
# create cursor 
cursor = db.cursor(pymysql.cursors.DictCursor)
db.autocommit(True)
# create table as per requirement
name = 'stephen'
age = 7
gradeLevel = 11
sql = "INSERT INTO students (name,age,gradeLevel) VALUES('stephen',7,11)"
sql2 = "SELECT * from students"
cursor.execute(sql)
cursor.execute(sql2)
# a cursor is predefined clas that can work with mysql

rows = cursor.fetchall()
for row in rows:
	print("name:" + row['name']+"" +" age:" + str(row['age']) + " " + " gradeLevel" + str(row['gradeLevel']))
cursor.close()
db.close()
